.travis.yml.example

```diff
@@ -0,0 +1,15 @@
+language: rust
+rust:
+  - 1.32.0
+cache: 
+  - rust
+  - cargo
+jobs:
+  include:
+    - stage: setup
+    - provider: script
+      script: bash ci.sh
```

Comment:
why introduce a new ci.sh script, can't we just run `setup.sh` and `build.sh` sequentially?

Reply:
Some of it was experimentation, but I've got it working now, this was because of me having issues with Travis and it taking so long to compile...